frontend/src/components/Blog/BlogCard.tsx

```
@@ -2,39 +2,40 @@ import React from "react";
-          <img src={imageUrl} alt={blog.title} className="w-full h-40 object-cover rounded-md" />
-        )}
-        <p className="line-clamp-3" dangerouslySetInnerHTML={{ __html: blog.content }}></p>
+      <CardContent className="flex flex-1 flex-col gap-4">
+        <div className="h-40 w-full overflow-hidden rounded-md bg-muted">
+          {imageUrl && (
+            <img src={imageUrl} alt={blog.title} className="h-full w-full object-cover" />
+          )}
+        </div>
+        <p
+          className="line-clamp-3 min-h-[4.5rem] text-sm leading-relaxed"
+          dangerouslySetInnerHTML={{ __html: blog.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)